app/design/frontend/base/default/template/esatisfaction/header.phtml

```diff
@@ -5,9 +5,24 @@ if ($status) {
     $applicationId = $helperData->getApplicationId();
     ?>
     <script>
-        window.esat_config =
-            {
-                application_id: '<?php echo $applicationId; ?>'
+        (function (win, doc, application_id, jq, collection) {
+            // Define e-satisfaction collection configuration
+            win.esat_config = {application_id: application_id, collection: collection || {}};
+
+            // Update metadata
+            win.Esat = win.Esat || {};
+            win.Esat.updateMetadata = function (questionnaireId, metadata) {
+                win.esat_config.collection[questionnaireId] = win.esat_config.collection[questionnaireId] || {};
+                win.esat_config.collection[questionnaireId].metadata = metadata;
             };
+
+            // Setup script
+            doc.addEventListener('DOMContentLoaded', function () {
+                var body = doc.getElementsByTagName('body')[0], script = doc.createElement('script');
+                script.async = true;
+                script.src = 'https://collection.e-satisfaction.com/dist/js/integration' + (!!jq ? '' : '.jq') + '.min.js';
+                body.appendChild(script);
+            });
+        })(window, document, "<?php echo $applicationId; ?>", false, {});

```

Comment:
There is an option (on deleted footer code) about jquery enabled. Use it here to define whether we need jQuery version or not.